controllers/mytba_controller.py

```diff
@@ -19,6 +21,7 @@ def get(self):
         self._require_registration('/account/register')
 
         user = self.user_bundle.account.key
+        now = datetime.datetime.now()
         team_favorites_future = Favorite.query(Favorite.model_type == ModelType.TEAM, ancestor=user).fetch_async()
 
         live_events = EventHelper.getWeekEvents()
```

Comment:
Perhaps change this to `getEventsWithinADay()` so a team doesn't show up in this list as well as the upcoming list? You might need to change the logic below to:

`e for e in events if e.start_date > now and not e.within_a_day()`